samples/SwiftDemoApp/SwiftDemoApp/Samples.swift

```diff
@@ -13,25 +13,25 @@
  * limitations under the License.
  */
 
-import Foundation
+import UIKit
 
 class Samples {
-  class func loadSamples() -> [Any] {
-    return [ newDemo(controller: ClusteringViewController.self,
+    class func loadSamples() -> [[String: Any]] {
+    return [ newDemo(controller: ClusteringViewController(),
                      title: "Clustering",
                      description: "Marker Clustering"),
-             newDemo(controller: KMLViewController.self,
+             newDemo(controller: KMLViewController(),
                      title: "KML",
                      description: "KML Rendering"),
-             newDemo(controller: GeoJSONViewController.self,
+             newDemo(controller: GeoJSONViewController(),
                      title: "GeoJSON",
                      description: "GeoJSON Rendering"),
-             newDemo(controller: HeatmapViewController.self,
+             newDemo(controller: HeatmapViewController(),
                      title: "Heatmaps",
                      description: "Heatmaps")]
   }
 
-    class func newDemo(controller: AnyClass!, title: String!, description: String!) -> [String : Any?] {
+    class func newDemo(controller: UIViewController, title: String, description: String) -> [String : Any] {
```

Comment:
the controller object holds in `[String: Any]` strongly.  So `AnyClass` just is a generic type that we ensure from keeping the instantiated object in a strong dictionary.

Reply:
Our intention is to get the View Controller, so why to write AnyClass or make it generic type?
IMO, Code will be more robust and readable if we specify the type rather than simply define AnyClass.

Reply:
Yes, `AnyClass` is protocol as you mentioned. so we can use `init()` constructor safely.